ENIGMAsystem/SHELL/Platforms/General/POSIX/Utility.cpp

```diff
@@ -1,8 +1,10 @@
 #include "Platforms/General/PFmain.h"
 
 #include <time.h> //CLOCK_MONOTONIC
+#ifndef _WIN32
```

Comment:
Why is this source being included on Win32? The name of this directory is POSIX. Are we doing that thing again where we pretend that MinGW=POSIX? Because we shouldn't be.